tools/nicythize

```diff
@@ -181,7 +181,7 @@ def find_process_files(root_dir):
     hash_db = load_hashes(HASH_FILE)
     for cur_dir, dirs, files in os.walk(root_dir):
         for filename in files:
-            for fromext, rule in rules.iteritems():
+            for fromext, rule in rules.items():
```

Comment:
If we needed the iterator, that would be better, but I think list (Py 2) or iterator (Py 3) is OK here.